new-lamassu-admin/src/pages/Blacklist/Blacklist.js

```diff
@@ -120,8 +121,20 @@ const Blacklist = () => {
     deleteEntry({ variables: { cryptoCode, address } })
   }
 
+  const validateAddress = (cryptoCode, address) => {
+    try {
+      return !R.isNil(coinUtils.parseUrl(cryptoCode, 'main', address))
+    } catch {
+      return false
+    }
+  }
+
```

Comment:
I think you can add this as a test on the yup schema, there's no need for it to be outside the modal.